app/src/main/java/com/github/siela1915/bootcamp/Labelling/AllergyType.java

```diff
@@ -35,6 +36,14 @@ public static AllergyType fromString(String text) {
         return null;
     }
 
+    public static AllergyType fromInt(Integer i){
+        return AllergyType.values()[i];
+    }
+
+    public static List<AllergyType> fromIntList(List<Integer> l){
+        return l.stream().map(i -> AllergyType.values()[i]).collect(Collectors.toList());
```

Comment:
It might be good practice to check that the input integers are within a valid range before accessing the array, same comment for the similar methods in `CuisineType` and `DietType`.